content/quizzes/en.yaml

```diff
@@ -1228,6 +1228,470 @@ levels:
             correct_answer: "B"
             hint: "Mind the order of the sentences."
             question_score: '10'
+    8:
+        1:
+            question_text: "Which output will be produced by this code?"
+            code: |-
+                {repeat} 2 {times}
+                    {print} 'Hello'
+                    {print} 'Im Hedy!'
+            mp_choice_options:
+                - "code": "Hello \n Im Hedy!"
```

Comment:
In de NL file heet deze tag option_text

Reply:
Ah I understand now, thanks for the clarification!! I will fix this in the nl version (and fix the rest fo the PR a bit too with the misplaced {})